server/controllers/user.controller.js

```diff
@@ -0,0 +1,24 @@
+const User = require("../models/user.model")
+
+const getUsersController = async function (req, res) {
+    try {
+        const usersFromDB = await User.find({})
```

Comment:
we don't need all users, this should be the sign in controller, where are you checking the password and the email